### ISSUE TITLE
SceneUnloadEndEventArgs doesn't contain unloaded scenes names

### ISSUE THREAD

**Issue by Maclay74** [FirstGearGames/FishNet#270]
Unity version: 2022.2
Fish-Networking version: 3.3.4

**Describe the bug**
When SceneManager fires `OnUnloadEnd` event, `Scene.name` is Null, making it not possible to know what scenes were unloaded.

**To Reproduce**
1. Subscribe to scene unload event on client
2. Unload client from a scene
3. Check scene name in argument 

**Expected behavior**
SceneUnloadEndEventArgs should contain scene names that were unloaded

**Comment by FirstGearGames** [comment]
Sorry to make you do this but can you update to 3.4.3 and confirm that the
issue still exist? We don't technically support older versions of the same
major release.

**Comment by FirstGearGames** [comment]
You may need to delete your fishnet folder before importing the new.

**Comment by Maclay74** [comment]
Okay, yes, I bumped to 3.4.3 and tried once again. Still Null here.

![image](https://user-images.githubusercontent.com/5504685/221371618-23d24bac-614f-4535-8940-eb7ab426102b.png)

**Comment by Maclay74** [comment]
Unfortunately, this one is a blocker for me, unless there is another way to learn unloaded scene's name.

**Comment by FirstGearGames** [comment]
I've not yet had a chance to look but I can assure you bugs will get
resolved.

**Comment by FirstGearGames** [comment]
This is actually a unity thing because the Scene name reference is lost after its unloaded. I'll add something to manually populate the info

**Comment by FirstGearGames** [comment]
I'll make the switch permanent in v4 but for now a solution is available in 3.5.3 (pending release).
```cs
        /// <summary>
        /// Unloaded scenes with names and handles cached.
        /// This will be renamed as UnloadedScenes in Fish-Networking version 4.
        /// </summary>
        public List<UnloadedScene> UnloadedScenesV2;
```

**Comment by Maclay74** [comment]
Thanks a lot!
